core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-reports/components/dot-experiments-reports-chart/dot-experiments-reports-chart.component.ts

```diff
@@ -29,27 +22,33 @@ import { getRandomUUID } from '../../../shared/dot-experiment.utils';
     changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class DotExperimentsReportsChartComponent implements OnChanges {
-    @ViewChild('chart') chart: UIChart;
-    options;
     @Input()
     isEmpty = true;
+
     @Input()
     isLoading = true;
+
     @Input()
     config: { xAxisLabel: string; yAxisLabel: string };
+
     @Input()
     data: ChartData<'line'>;
 
-    protected chartId = `chart-` + getRandomUUID();
+    @Input()
+    isLinearAxis = false;
 
+    options;
```

Comment:
do we need a type for this?

Reply:
I was looking into primeng and they didn't typed... chartjs have types tho... but :(

Reply:
Let me try.